src/Views/gagepage.html

```diff
@@ -239,7 +239,45 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                         </div>
                         <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.heatChartConfig">
                             <highchart id="chart2" config="vm.heatChartConfig"></highchart>
-                        </center>
+                        </div>
+                    </center> -->
+                    <center>
+                        <div ng-if="!vm.dischargeChartConfig" class="panel panel-default reportSpinner">
+                            &nbsp;<i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i>
+                        </div>
+                        <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.dischargeChartConfig">
+                        <input type="checkbox" id="logScaleDischarge" name="logScaleDischarge" ng-model="vm.logScaleDischarge" ng-change="vm.toggleLogLinearDischarge()">
+                        <label for="logScaleDischarge" > Log Scale</label>
+                        
+                        <input type="radio" id="ageData" name="ageQualityData" ng-model="vm.ageQualityData" value="age" ng-change="vm.toggleDischargeData('age')"> 
```

Comment:
The Age and Quality radio buttons should not appear if there are no USGS Measurements on the graph

Reply:
Like Harper's comment if there are not any USGS Measured value, get rid of it on the legend, ran into a similiar problem where I couldn't just show button if length was > 0 since all my color values were already in the function with or without data.

So like the one for removing it form the legend, I solved this one likely in a not consistent way. It works for the most part, but will need to find a more consistent way sometime.